pegasus/data/cdo-glossary-csf.csv

```diff
@@ -28,7 +28,8 @@ call,call (a function),This is the piece of code that you add to a program to in
 click,click,Press the mouse button.,,,,,,,,,,,,
 code,code,The language that programmers create and use to tell a computer what to do.,Course 1: Stage 2,Course 1: Stage 2,,,,,,,,,,
 command,command,An instruction for the computer. Many commands put together make up algorithms and computer programs.,Course 1: Stage 2,Course 1: Stage 2,,,,,,,,,,
-computational thinking,computational thinking,"Mental processes and strategies that include: decomposition, pattern matching, abstraction, algorithms (decomposing problems into smaller, more manageable problems, finding repeating patterns, abstracting specific differences to make one solution work for multiple problems, and creating step-by-step algorithms).",Course 3: Stage 1,Course 3: Stage 1,,,,,,,,,,
+computational thinking,computational thinking,"Modifying a problem in such a way that it can be modeled or solved using a computer or machine.
```

Comment:
we shouldn't have changes to these csv's in PR, they'll get overwritten by gsheet changes once merged. Let's not add them to commits.